src/plugins/dashboard/public/application/top_nav/editor_menu.tsx

```diff
@@ -51,7 +51,7 @@ export const EditorMenu = ({ dashboardContainer, createNewVisType }: Props) => {
 
   const trackUiMetric = usageCollection?.reportUiCounter.bind(
     usageCollection,
-    DashboardConstants.DASHBOARDS_ID
```

Comment:
@ThomThomson it looks like a typo. `dashboard` should be here not a `dashboards`

Reply:
Hum, `dashboards` is the app id, and it's what is used to navigate in this same file. Though it looks like `dashboard` is what is used in the `kibana.json`. I might look into unifying this, but in the scope of this PR, you could probably use either!